zerver/tests/test_messages.py

```diff
@@ -587,6 +591,82 @@ def test_applying_markdown(self):
         self.assertEqual(message.rendered_content, expected_content)
         self.assertEqual(message.rendered_content_version, bugdown.version)
 
+
+    def test_reaction(self):
```

Comment:
That makes perfect sense. Where would the integration tests go? In `test_integrations.py`?